ribbon/src/main/java/feign/ribbon/LBClient.java

```diff
@@ -99,8 +99,12 @@ Request toRequest() {
           .request();
     }
 
+    Client client() {
```

Comment:
ps why was this change made? (from field on LBClient to field RibbonRequest) I may have lost it during the review.